go.mod

```diff
@@ -35,15 +35,16 @@ require (
 	go.opentelemetry.io/otel/trace v1.4.0
 	golang.org/x/crypto v0.1.0
 	golang.org/x/mod v0.6.0-dev.0.20220419223038-86c51ed26bb4
-	golang.org/x/text v0.4.0
+	golang.org/x/text v0.5.0
```

Comment:
I am not so sure if minor version upgrade for `golang.org/x/text` is needed here?

Reply:
I'm guessing it came with the cloud-provider dependency, @chiukapoor to confirm

Reply:
Yes, it came with the `cloud-provider-azure` dependency

https://github.com/kubernetes-sigs/cloud-provider-azure/blob/bf1d7f3b46068081d795f30d440ef0997a3e9209/go.mod#L23